lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -121,6 +122,30 @@
         "time": { "type": "string", "format": "datetime" }
       }
     },
+    "account": {
+      "type": "object",
+      "description": "Represents a change to an account's status on a host (eg, PDS or Relay). The semantics of this event are that the status is at the host which emitted the event, not necessarily that at the currently active PDS. Eg, a Relay takedown would emit a takedown with active=false, even if the PDS is still active.",
+      "required": ["seq", "did", "time", "active"],
+      "properties": {
+        "seq": { "type": "integer" },
+        "did": { "type": "string", "format": "did" },
+        "time": { "type": "string", "format": "datetime" },
+        "active": {
+          "type": "boolean",
+          "description": "Indicates that the account has a repository which can be fetched from the host that emitted this event."
+        },
+        "status": {
+          "type": "string",
```

Comment:
made a tweak here

with token values we normally just use a `string` type & list the token values in `knownValues`.

see `com.atproto.moderation.defs` for an example with report types

Reply:
To be honest I am not even sure if we need tokens here— its possible it could be non-namespaced strings.

In my mind the value in tokens is that it allows extensibility by other namespaces, especially in cases where it's friendly for applications to be able to use values that will never make it into the lexicon's `knownValues`.  I don't think that's necessarily needed or even desirable here in the sync endpoints.  One reason may not want that, is that it's nice to keep the values one-to-one with the error types on these sync endpoints, which are not namespaced.  That allows you to understand the repo's status fully through the response of e.g. `sync.getRepo`.

Reply:
That's a good point - and I do tend to agree that this isn't a place that we're asking for extensibility on. Any changes should be part of the formal sync protocol 🤔 

So maybe we just do hardcoded values?

Reply:
That would be my vote!  I still support keeping it an open union as a little safety net.

Reply:
it feels like a place "hardcoded" would make sense. agree on keeping it open-union semantics.